Corrected 1V/1A to give 1 ohm

RituRajSingh878 opened the pull request:
<!-- Your title above should be a short description of what
was changed. Do not include the issue number in the title. -->

#### References to other Issues or PRs
<!-- If this pull request fixes an issue, write "Fixes #NNNN" in that exact
format, e.g. "Fixes #1234". See
https://github.com/blog/1506-closing-issues-via-pull-requests .-->
Fixes  #15647".

#### Brief description of what is fixed or changed
ohm.scale_factor=1000


This value of 1000 conflicts with the definition, this should be 1 .
and the reason of this is "scale_factor"  of "kg" is not one.
so i changed "definitions" file to make it 1.
and also test file. 

#### Other comments


#### Release Notes

<!-- Write the release notes for this release below. See
https://github.com/sympy/sympy/wiki/Writing-Release-Notes for more information
on how to write release notes. The bot will check your release notes
automatically to see if they are formatted correctly. -->

<!-- BEGIN RELEASE NOTES -->
- physics.units
  - corrected scale factor Kilogram to 1 as per SI unit.


  <!-- END RELEASE NOTES -->

RituRajSingh878 committed:
add test for numpy printer

RituRajSingh878 committed:
update pretty printer for physics vector

RituRajSingh878 committed:
done

RituRajSingh878 committed:
changing scale_factor of kg to "One"

SI unit of mass is "kg" so scaling factor of "kg" should be "one"

RituRajSingh878 committed:
update for SI unit of mass.

As SI unit of mass is taken as "kg" .

RituRajSingh878 committed:
delete

RituRajSingh878 committed:
done

RituRajSingh878 committed:
done

RituRajSingh878 committed:
nothing changes

RituRajSingh878 committed:
changing scale_factor of kg to "One"


changing scale_factor of kg to "One"

SI unit of mass is "kg" so scaling factor of "kg" should be "one"

RituRajSingh878 committed:
update for SI unit of mass.

RituRajSingh878 committed:
done

smichr commented on sympy/physics/units/definitions.py:
```diff
@@ -53,7 +53,7 @@
-gram.set_scale_factor(One)
+gram.set_scale_factor(Rational(1, 1000))
 
 # NOTE: the `kilogram` has scale factor 1000. In SI, kg is a base unit, but
```
Does this note still apply?

jksuom replied:
According to [this comment](https://github.com/sympy/sympy/pull/15664#issuecomment-449357217) it seems to apply.

smichr commented on sympy/physics/units/definitions.py:
```diff
@@ -53,7 +53,7 @@
 g = gram = grams = Quantity("gram", abbrev="g")
 gram.set_dimension(mass)
-gram.set_scale_factor(One)
+gram.set_scale_factor(Rational(1, 1000))
```
Could this be `kg/kilo`? That would require defining kg first, however, right?

RituRajSingh878 replied:
it seems right to me.

smichr committed:
use kg/kilo instead of hardcoded 1/1000

smichr commented:
@RituRajSingh878 , I made a few mods. If those are agreeable to you then would you mind squashing all the commits into 1 and repushing the changes here? `git push -f RituRajSingh878 SiUnit`

RituRajSingh878 committed:
changing scale_factor of kg to "One"

SI unit of mass is "kg" so scaling factor of "kg" should be "one"
update for SI unit of mass.

As SI unit of mass is taken as "kg" .
delete
done
done
nothing changes
 changing scale_factor of kg to "One"


changing scale_factor of kg to "One"

SI unit of mass is "kg" so scaling factor of "kg" should be "one"
 update for SI unit of mass.
done
use kg/kilo instead of hardcoded 1/1000

RituRajSingh878 committed:
Merge remote-tracking branch 'origin/SiUnit' into SiUnit

RituRajSingh878 committed:
changing scale_factor of kg to One
